exchanges/interfaces.go

```diff
@@ -53,6 +53,7 @@ type IBotExchange interface {
 	SubmitOrder(s *order.Submit) (order.SubmitResponse, error)
 	ModifyOrder(action *order.Modify) (string, error)
 	CancelOrder(o *order.Cancel) error
+	CancelBatchOrders(o *order.Cancel) (order.CancelBatchResponse, error)
```

Comment:
I think it would be best to update the function signature to be `o []order.Cancel` instead. There is a lack of comments in the implemtentation functions to denote that a user must use a comma delimited string to cancel multiple orders. It also would allow for more flexibility in future for cancellations eg a SPOT and FUTURES cancellation batch request

Reply:
done